praw/models/reddit/base.py

```diff
@@ -62,46 +80,50 @@ def __init__(self, reddit, _data):
         :param reddit: An instance of :class:`~.Reddit`.
 
         """
-        super(RedditBase, self).__init__(reddit, _data)
+        super(RedditBase, self).__init__(reddit, _data=_data)
         self._fetched = False
         self._info_params = {}
 
     def __repr__(self):
-        """Return an object initialization representation of the instance."""
-        return "{}({}={!r})".format(
-            self.__class__.__name__, self.STR_FIELD, str(self)
+        """Return a representation of the instance."""
+        return "<{}({}={!r})>".format(
```

Comment:
I now understand your reason for changing the format and you have a good point. However, I'd consider this to be a breaking change, so I'd like to hold off on this and similar changes until we actually introduce the breaking change.

Reply:
This PR doesn’t intend to be backwards compatible. The very first commit makes a breaking change. While it’s avoidable, the time needed to implement it in a backwards compatible way probably isn’t worth it.

I suggest a new PR be made if you wish for features of this PR to be back-ported.